x/axelarnet/keeper/genesis.go

```diff
@@ -1,14 +1,21 @@
 package keeper
 
 import (
+	"bytes"
+
 	"github.com/axelarnetwork/axelar-core/x/axelarnet/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 // InitGenesis initializes the reward module's state from a given genesis state.
 func (k Keeper) InitGenesis(ctx sdk.Context, nexus types.Nexus, genState *types.GenesisState) {
 	k.setParams(ctx, nexus, genState.Params)
-	k.SetFeeCollector(ctx, genState.CollectorAddress)
+	if genState.CollectorAddress != nil && !bytes.Equal(genState.CollectorAddress.Bytes(), []byte{}) {
```

Comment:
Why? `sdk.VerifyAddressFormat` is already doing 
```
        if len(bz) == 0 {
		return sdkerrors.Wrap(sdkerrors.ErrUnknownAddress, "addresses cannot be empty")
	}
```

Reply:
Because we do not want to force the genesis file to always include a fee collector. And to do that, we need to explicitly verify that it is neither nil, nor a byte slice with length zero. Given that the genesis file contains the `fee_collector` field set to `""`, we will get a byte slice with length zero.

Reply:
ok then just 
```
if len(genState.CollectorAddress) > 0 {
```

Reply:
done